src/components/filter/filter.css

```diff
@@ -0,0 +1,30 @@
+.filter-container {
+  display: flex;
+  padding: 20px;
+  margin: 10px;
+  align-items: center;
+  justify-content: center;
+}
+.filter-container__list {
+  display: flex;
+  gap: 20px;
+  list-style: none;
+}
+.filter-container__list-item {
```

Comment:
what do you think about adding a hover effect? like a new background-color

Reply:
Good! Added hover effect:

https://user-images.githubusercontent.com/42494406/155044817-71f29efd-d34c-4f71-9f7c-235daeafe4bd.mov